src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <title>Javascript Group Capstone Project By David Yannick and Saningo  </title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <a class="link-logo" href="./index.html">Logo</a>
+      <nav class="links">
+        <ul class="link-lists">
+          <li class="link-list"><a href="#" >About</a></li>
+          <li class="link-list"><a href="#" >Contact</a></li>
+          <li class="link-list"><a href="./index.html" class="link tv-show-link">.</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main-content">
+    <div class="grid-cards-container"></div>
+  </main>
```

Comment:
- [ ] Nice job adding the reservations section as requested by the previous reviewer. However, the change has caused another issue. I mean you have the same page for comments and reservations

https://user-images.githubusercontent.com/67087664/164895837-867d4673-bb25-4cd8-9168-4f263ec9f41c.mp4

Kindly make pages separately for comments and reservations not the same page. for example, write comments instead of reservations on the comments page